velox/docs/develop/connectors.rst

```diff
@@ -0,0 +1,39 @@
+==========
+Connectors
+==========
+
+Connectors allow Velox to read and write data to and from external sources.
+This concept is similar to `Presto Connectors <https://prestodb.io/docs/current/develop/connectors.html>`_.
+The :ref:`TableScanNode<TableScanNode>` operator reads data from a connector.
+The various connector interfaces in Velox are described below.
+
+Connector Interface
+-------------------
+
+.. list-table::
+   :widths: 10 40
+   :header-rows: 1
+
+   * - Interface Name
+     - Description
+   * - ConnectorSplit
+     - A split is the basic unit that describes a chunk of data.
+   * - DataSource
+     - Provides methods to consume and process a split. A DataSource can optionally consume a
+       dynamic filter during execution to prune some rows from the output vector.
+   * - DataSink
+     - Provides methods to write a Velox vector externally.
+   * - ConnectorQueryCtx
+     - Interface to specify configuration to a DataSink or a DataSource.
+   * - Connector
+     - This is the main interface that allows creating instances of a DataSource or a DataSink.
+   * - Connector Factory
+     - This interface enables creating instances of connectors using the specified name.
```

Comment:
It would be nice to clarify the difference between connector name and ID.

Reply:
done